### ISSUE TITLE
Comments do not indent when using "comment with Wordpress and Google accounts" in the Papr premium theme

### ISSUE THREAD

**Issue by matteo-greco** [Automattic/jetpack#17524]
As a followup to a conversation with support, we were asked to open a bug report here.
Unfortunately, the theme authors declided to cooperate on this, so I (the client's web developer) am opening it on their behalf.
I realize the following information may not be enough and you may not have access to the theme, so let me know what I can do to make fixing this easier.

#### Steps to reproduce the issue

1. Activate the "comment with Wordpress and Google accounts" option
2. Add a comment and a reply to said comment
3. Turn off "comment with Wordpress and Google accounts"
4. Add another reply to the first comment

#### What I expected

Both reply comments should indent.

#### What happened instead

The first reply comment does not indent, while the second reply comment does.

Here's a screenshot 
![Screenshot_1](https://user-images.githubusercontent.com/20144092/96456362-adda0f00-121e-11eb-9fd7-d88511f182df.jpg)

**Comment by jeherve** [comment]
Could you check the theme's files, especially `functions.php`, and check that the `comment-reply` script is being enqueued like so:
https://github.com/WordPress/twentytwenty/blob/6d0a5240af108d02f58ec797fd49b32458a4c698/functions.php#L204-L206

If not in `functions.php`, it may be in a different file that is being included from there.

Let us know what you find.

**Comment by matteo-greco** [comment]
The only reference to `comment-reply` is this:

```php
if (is_singular() && comments_open() && get_option('thread_comments')) {
    wp_enqueue_script('comment-reply');
}
```

**Comment by jeherve** [comment]
Thank you! Do you also see the script being loaded on the site, if you check your browser's network tools?

![image](https://user-images.githubusercontent.com/426388/96492810-7aaa7680-1244-11eb-9a50-572fc24854a8.png)

**Comment by matteo-greco** [comment]
Yes, I can confirm that comment-reply.js is loaded (200 ok)
It's the .min version and it was served from cache, but I don't think that makes any difference, right?

![Screenshot_2](https://user-images.githubusercontent.com/20144092/96559003-c7319880-12bc-11eb-9931-62b89a454fbf.jpg)

**Comment by jeherve** [comment]
Thanks for checking! I'll reach out to you via email again to see if we can debug this further.

**Comment by jeherve** [comment]
Internal reference: 3423281-zen

**Comment by matteo-greco** [comment]
Thank you, I'll wait for your e-mail

**Comment by matteo-greco** [comment]
Hi, I still haven't heard from support. Have you tried to get in touch with me?

**Comment by jeherve** [comment]
Yes, I emailed you on October 20. I'll send you the email again.

**Comment by matteo-greco** [comment]
I have not received any e-mail, I think you may be sending it to a different address.
How can I send you my address privately?

**Comment by jeherve** [comment]
@matteo-greco I've been sending those emails to the address we've used in previous email exchanges. 

> How can I send you my address privately?

Could you contact us via [this contact form](https://jetpack.com/contact-support/?rel=support) and mention this GitHub issue? I'll get back to you!

Thank you.

**Comment by matteo-greco** [comment]
> Could you contact us via this contact form and mention this GitHub issue? I'll get back to you!

I tried, but it seems to require a paid Jetpack plan, which my client has but I don't. Any other way?

**Comment by jeherve** [comment]
> Any other way?

Could you reply to one of the emails you received from us, from the past conversations you've had with our support team?

Thank you.

**Comment by matteo-greco** [comment]
I wrote to the support@ e-mail address. Is that alright?

**Comment by htdat** [comment]
> I wrote to the support@ e-mail address. Is that alright?

@matteo-greco - That's right. We've replied to you. 

@jeherve -  Intertal ref: 3423281-zen

**Comment by matteo-greco** [comment]
Thank you, I'm now in touch with your support via e-mail.
